### PR TITLE
openfortivpn: add page

### PR DESCRIPTION
**Pull request by robbyrenz**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

I decided to add a man page for a terminal-based VPN program that I use to tunnel to a VPN.

### CODE REVIEW

File: pages/linux/openfortivpn.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+> Client that connects to a VPN.
+
+- Connect to a VPN with a username and password:
+
+`openfortivpn -u {{username}} -p {{password}}`
+
+- Connect to a VPN defined in a {{/etc/openfortivpn/config}} file:
+
+`sudo openfortivpn -c config`
```
Comment (navarroaxel) [inline pages/linux/openfortivpn.md:11]: The `--config` is visible in `--help`:
```suggestion
`sudo openfortivpn --config=path/to/config`
```

File: pages/linux/openfortivpn.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# openfortivpn
+
+> Client that connects to a VPN.
+
+- Connect to a VPN with a username and password:
+
+`openfortivpn -u {{username}} -p {{password}}`
+
+- Connect to a VPN defined in a {{/etc/openfortivpn/config}} file:
```
Comment (navarroaxel) [inline pages/linux/openfortivpn.md:9]:
```suggestion
- Connect to a VPN using a specific configuration file (defaults to `/etc/openfortivpn/config`):
```

File: pages/linux/openfortivpn.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# openfortivpn
+
+> Client that connects to a VPN.
+
+- Connect to a VPN with a username and password:
+
+`openfortivpn -u {{username}} -p {{password}}`
```
Comment (navarroaxel) [inline pages/linux/openfortivpn.md:7]: We prefer the long argument format:
```suggestion
`openfortivpn --username={{username}} --password={{password}}`
```

File: pages/linux/openfortivpn.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# openfortivpn
+
+> Client that connects to a VPN.
```
Comment (navarroaxel) [inline pages/linux/openfortivpn.md:3]: Based on `openconnect` for Cisco:
```suggestion
> A VPN client, for Fortinet's proprietary PPP+SSL VPN solution.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @robbyrenz, I left some comments.

I think you could add some extra examples using a `host` and `port` in the CLI, and another for `trusted-cert`

**Comment by robbyrenz** [comment]
Thanks for the feedback! I have addressed all of them. Hope you can take a look again :smiley:

File: pages/linux/openfortivpn.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# openfortivpn
+
+> A VPN client, for Fortinet's proprietary PPP+SSL VPN solution.
```
Comment (navarroaxel) [inline pages/linux/openfortivpn.md:3]: Oh, I found the repo:
```suggestion
> A VPN client, for Fortinet's proprietary PPP+SSL VPN solution.
> More information: <https://github.com/adrienverge/openfortivpn>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @robbyrenz just add the page link.

File: pages/linux/openfortivpn.md (hunk 5 above)
Comment (robbyrenz) [reply pages/linux/openfortivpn.md:3]: Thanks for the link! I added it :+1:

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/openfortivpn.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://github.com/adrienverge/openfortivpn>.
+
+- Connect to a VPN with a username and password:
+
+`openfortivpn --username={{username}} --password={{password}}`
+
+- Connect to a VPN using a specific configuration file (defaults to `/etc/openfortivpn/config`):
+
+`sudo openfortivpn --config=path/to/config`
```
Comment (sbrl) [inline pages/linux/openfortivpn.md:12]: Perhaps this will read better?
```suggestion
`sudo openfortivpn --config={{path/to/config}}`
```

**Review by owenvoke — APPROVED** [review]
LGTM after @sbrl's change has been applied.

**Pull request merged**